ev3dev/button.py

```diff
@@ -28,14 +28,283 @@
 if sys.version_info < (3,4):
     raise SystemError('Must be using Python 3.4 or higher')
 
+import array
+import time
+import evdev
```

Comment:
This is probably a linux-specific module like fcntl

Reply:
I was meaning to ask about the try/except around fcntl...do we have folks running `api_test.py` on windows?

Reply:
Probably not, but any editor with code completion will load the modules.